solr/solrj/src/test/org/apache/solr/client/solrj/response/TestSuggesterResponse.java

```diff
@@ -119,8 +119,8 @@ public void testEmptySuggesterResponse() throws Exception {
   }
 
   private void addSampleDocs() throws SolrServerException, IOException {
-    getClient().deleteByQuery("*:*");
-    getClient().commit(true, true);
+    solrClientTestRule.getSolrClient().deleteByQuery("*:*");
+    solrClientTestRule.getSolrClient().commit(true, true);
```

Comment:
Nope; we agreed we'd leave much test code as it was, simply calling `getSolrClient` defined by SolrJettyTestBase which will indeed get the client from the rule.